### ISSUE TITLE
Force flag is missing in volume delete

### ISSUE THREAD

**Issue by Shrivaibavi** [gluster/glusterd2#1205]
``` [root@dhcp35-179 glusterd2]# glustercli volume delete vol1 --force
Error: unknown flag: --force
Usage:
  glustercli volume delete [flags]

Flags:
  -h, --help   help for delete

Global Flags:
      --cacert string        Path to CA certificate
      --endpoints strings    glusterd2 endpoints (default [http://127.0.0.1:24007])
      --insecure             Accepts any certificate presented by the server and any host name in that certificate.
      --json                 JSON Output
      --loglevel string      Severity of messages to be logged (default "INFO")
  -y, --script-mode          running in script mode
      --secret string        Password for authentication
      --secret-file string   Path to file which contains the secret for authentication
      --timeout uint         overall client timeout (in seconds) which includes time taken to read the response body (default 30)
      --user string          Username for authentication (default "glustercli")
  -v, --verbose              verbose output
      --xml                  XML Output

unknown flag: --force
```

**Comment by rishubhjain** [comment]
@Shrivaibavi  I think we don't support a force option in volume delete. why is it required?

**Comment by oshankkumar** [comment]
@Shrivaibavi if you want to disable the prompt while deleting volume you can give -y or --script-mode flag

**Comment by Madhu-1** [comment]
the original concern to ask to ask for `force` flag is
 in the below scenario we are not able to delete the volume as the directory is not present

if we provide the `force` flag we can continue to delete volume  if we get some error like `not found`

``` [root@dhcp35-179 ~]# glustercli volume delete vol1
Are you sure you want to delete volume vol1 [yes/no]? yes
Volume deletion failed

Response headers:
X-Request-Id: 93e994c2-e739-4045-91eb-f60a07eea199
X-Gluster-Cluster-Id: f010c420-2797-46e5-b249-098eb1ec50f1
X-Gluster-Peer-Id: 3c23daaf-f343-4ac8-bd3f-186a44101df2

Response body:
Transaction step vol-delete.CleanBricks failed on peer 3c23daaf-f343-4ac8-bd3f-186a44101df2 with error: lstat /run/glusterd2/snaps/vol1: no such file or directory
Transaction step vol-delete.CleanBricks failed on peer 8fa3e7e6-a5ac-47b2-9ba3-ca29997dbeb0 with error: lstat /run/glusterd2/snaps/vol1: no such file or directory
Transaction step vol-delete.CleanBricks failed on peer d112190f-5213-417f-83c6-4ba045ead1c4 with error: lstat /run/glusterd2/snaps/vol1: no such file or directory
```

@aravindavk @prashanthpai need your thoughts on this

**Comment by aravindavk** [comment]
@Madhu-1 I debugged the issue, that issue was due to volume snapshot was created before this patch https://github.com/gluster/glusterd2/pull/1178

Clone volume from previously created snapshot was failing because of upgrade. So cleaning etcd store and fresh setup may not have this issue. @Shrivaibavi will open new issue if the issue mentioned by you found again.

What are the expectations from `--force` flag? Delete to succeed even if there are some failures on some nodes or ignore all failures.
